pkg/agent/rest.go

```diff
@@ -30,31 +30,26 @@ type NodeZeroRestClient struct {
 func NewNodeZeroRestClient(ctx context.Context, assetDir string) (*NodeZeroRestClient, error) {
 	restClient := &NodeZeroRestClient{}
 
-	assetStore, err := assetstore.NewStore(assetDir)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to create asset store")
-	}
-	nmState := &manifests.NMStateConfig{}
-	if err := assetStore.Fetch(nmState); err != nil {
-		return nil, errors.Wrapf(err, "failed to fetch %s", nmState.Name())
-	}
+	agentManifests := &manifests.AgentManifests{}
+	agentConfigAsset := &agentconfig.AgentConfig{}
```

Comment:
These are just going to be empty structures unless we call `assetStore.Fetch()` to load them I think.